custom_model_runner/datarobot_drum/drum/language_predictors/python_predictor/python_predictor.py

```diff
@@ -63,3 +50,17 @@ def predict(self, input_filename):
         self.monitor(input_filename, predictions, execution_time_ms)
 
         return predictions
+
+    def predict_unstructured(self, **kwargs):
+        # validating_input
+        data_text = kwargs.get("text", None)
```

Comment:
Line of code that I referred to that has both text/data. Commenting since we agreed to only use data.

Reply:
fixed